charts/celeborn/templates/_helpers.tpl

```diff
@@ -47,25 +47,25 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Selector labels
+*/}}
+{{- define "celeborn.selectorLabels" -}}
+app.kubernetes.io/name: {{ .Chart.Name }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/tag: {{ .Values.image.tag }}
+{{- end }}
+
 {{/*
 Common labels
 */}}
 {{- define "celeborn.labels" -}}
-helm.sh/chart: {{ include "celeborn.chart" . }}
+helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
```

Comment:
It's not the common labels include selectorLabels, but selectorLabels should contains Common labels？